atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/ThreeNodeCassandraCluster.java

```diff
@@ -65,6 +66,15 @@
             .leaders(ImmutableSet.of("localhost"))
             .build());
 
+    private final String cassandraVersion;
```

Comment:
You should be able to make this a constant that pulls in the environment variable.